bundles/org.openhab.io.homekit/README.md

```diff
@@ -560,50 +560,50 @@ Number          thermostat_target_temp     "Thermostat Target Temp[%.1f °C]"
 
 If "useFahrenheitTemperature" is set to true, the min and max temperature must be provided in Fahrenheit.
 
-#### Thermostat modes
+#### Thermostat Modes
 
-HomeKit thermostat supports following modes
+The HomeKit thermostat supports the following modes:
```

Comment:
This reads a little weird. "HomeKit thermostats support" instead?

Reply:
There is only one homekit thermostat accessory, right? 
It is not about _all_ thermostats.